home/models.py

```diff
@@ -30,3 +30,31 @@ class Profile(models.Model):
 
     def __str__(self):
         return '{self.user.username}'.format(self=self)
+
+
+class Subject(models.Model):
+    subject_name = models.CharField(max_length=100, unique=True)
+
+    def __str__(self):
+        return self.subject_name
+
+
+class Sub_Subject(models.Model):
```

Comment:
The standard for Python class names is `CamelCase`. `snake_case` is used for function, method and property names. I don't think I've seen such a mixed name form used anywhere...